api/src/main/java/jakarta/servlet/http/Cookie.java

```diff
@@ -422,4 +423,101 @@ public void setHttpOnly(boolean isHttpOnly) {
     public boolean isHttpOnly() {
         return isHttpOnly;
     }
+
+    /**
+     * Sets the <i>SameSite</i> attribute for this cookie with the provided value.
+     *
+     * <p>
+     * When <code>sameSite</code> is not <code>null</code>, this cookie will have its <code>SameSite</code>
+     * attribute set with the provided value.
+     *
+     * @param sameSite the <i>SameSite</i> attribute's value; may be null if the attribute should not be set
```

Comment:
I think that we need to allow for a container default:
```java

￼     /**
￼      * Sets the <i>SameSite</i> attribute for this cookie with the provided value.
￼      *
￼      * <p>
￼      * When <code>sameSite</code> is not <code>null</code>, this cookie will have its <code>SameSite</code>
￼      * attribute set with the provided value.  When <code>sameSite</code> is <code>null</code>, the container may 
      * use a context specific default for the cookie's <code>SameSite</code> attribute or it may not be set.
￼      *
￼      * @param sameSite the <i>SameSite</i> attribute's value; may be null if the attribute should not be a
      * container provided context default or not set if there is no context default.
      */
```

Reply:
I've updated the JavaDocs as requested.
One thing to note, is that I think there was a typo in the @param definition:

> may be null if the attribute **should not be** a container provided context default or not set if there is no context default.

I think it was meant to be
> may be null if the attribute **should be** ...